scripts/e2e-test.sh

```
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# E2E Test Helper Script
+# This script starts all required services and runs E2E tests
+
+set -e
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.11.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)